omod/src/main/resources/config.xml

```diff
@@ -67,7 +67,11 @@
 	<filter-mapping>
 		<filter-name>clientAuthenticationFilter</filter-name>
 		<url-pattern>/keycloak/*</url-pattern>
```

Comment:
You definitely don't need this URL pattern. A URL pattern is just matched against the incoming URL without the host (http://localhost) or context (/openmrs), and OpenMRS doesn't have anything that responds to /keycloak.